.changeset/automated-from-21eae416d9fa85ea6f9be55945c44ab3da2c655b.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
I'm not sure how changeset is going to handle these package-less entries :sweat_smile: 
we'll see when https://github.com/embroider-build/embroider/pull/1404 updates!